sampleapp/templates/account/password_reset.jinja

```diff
@@ -0,0 +1,30 @@
+{% from 'django/forms/inputgroup.jinja' import inputgroup %}
+{% from 'components/button.jinja' import button %}
+{% extends "account/account_base.jinja" %}
+{% block account_title %}
+    {% trans %}
+        Reset Password
+    {% endtrans %}
+{% endblock account_title %}
+{% block account_content %}
+    {% if request.user.is_authenticated %}
+        {% include "account/snippets/already_logged_in.jinja" %}
+    {% endif %}
+    <p>
+        {% trans %}
+            Forgotten your password? Enter your e-mail address below, and we'll send you an e-mail allowing you to reset it.
+        {% endtrans %}
+    </p>
+    <form method="post"
+          action="{{ url('account_reset_password') }}"
+          class="password_reset">
+        {% csrf_token %}
+        {{ inputgroup(form) }}
```

Comment:
I think this should just be {{form}} - inputgroup is for a single field/lable/error